js/react_native/.gitignore

```diff
@@ -0,0 +1,68 @@
+# OSX
```

Comment:
we can keep all filenames and folders as kebabCase since this project contains iOS and Android native codes, which has their own convention. Do you have any specific filename or folder that needs to be fixed?